### ISSUE TITLE
Comments lists are not responsive to container width

### ISSUE THREAD

**Issue by kmshin1397** [skyportal/skyportal#1613]
The comments list container does not expand to fill the width of its parent Accordion component

**Comment by dmitryduev** [comment]
@kmshin1397 ping?

**Comment by kmshin1397** [comment]
Ah yes still planning to get to this at some point just forgot about it since it was low priority

**Comment by joelachance** [comment]
Looking at this one, here's some changes:

1. I removed the background color for non-hover states. There's only a background when hovered over.
2. Removed some padding to fit some more comments in the space we have
3. Updated the delete button to fit closer with the compact comments ticket #976 

<img width="446" alt="Screen Shot 2021-05-18 at 10 49 36 PM" src="https://user-images.githubusercontent.com/3891660/118753972-a833be80-b82b-11eb-9829-ac41c4b89d38.png">
<img width="437" alt="Screen Shot 2021-05-18 at 10 56 11 PM" src="https://user-images.githubusercontent.com/3891660/118754260-44f65c00-b82c-11eb-8cf4-214010675589.png">
